clients/src/main/java/org/apache/kafka/common/utils/Checksums.java

```diff
@@ -30,6 +33,18 @@
  */
 public final class Checksums {
 
+    private static final MethodHandle BYTE_BUFFER_UPDATE;
+
+    static {
+        MethodHandle byteBufferUpdate = null;
+        try {
+            byteBufferUpdate = MethodHandles.publicLookup().findVirtual(Checksum.class, "update", MethodType.methodType(void.class, ByteBuffer.class));
+        } catch (Throwable silent) {
```

Comment:
I've added a LOG::warn in case the given signature is expected to be found (although it doesn't consider the security manager enabling option - but I don't believe is a common Kafka practice suggested to users).